src/coreclr/tools/Common/TypeSystem/Common/MethodDesc.cs

```diff
@@ -143,7 +143,7 @@ public bool IsAsyncCallConv
         {
             get
             {
-                return (_flags & MethodSignatureFlags.AsyncCallConv) != 0;
+                return (_flags & MethodSignatureFlags.AsyncCallingConvention) != 0;
```

Comment:
It looks a bit odd for the internal bit to be `AsyncCallingConvention` and the property to be called `IsAsyncCallConv`.

The pre-existing names are variants of AsyncCall (`isAsyncCall` and `CORINFO_CALLCONV_ASYNCCALL`), I would use the same name here as well: `IsAsyncCall` for the property and `MethodSignatureFlags.AsyncCall` for the bit.